stencil_main.py

```diff
@@ -180,31 +140,25 @@ def main(
         "test_gt4py",
     ]
     if stencil_name not in stencil_name_list:
-        print(
+        raise ValueError(
             "please make sure you choose one of the following stencil: {}".format(
                 stencil_name_list
             )
         )
-        sys.exit(0)

```

Comment:
Raising an exception is the standard way to handle undesired situations in Python.